Sources/SwiftProtobuf/MathUtils.swift

```diff
@@ -43,46 +43,46 @@ internal func div<T : SignedInteger>(_ a: T, _ b: T) -> T {
 //
 // Swift 3 called this initializer "truncatingBitPattern";
 // Swift 4 (prerelease) changed this initializer to
```

Comment:
Maybe drop the "(prerelease)" part since I doubt it is changing again?

Reply:
Done.